internify/src/components/molecules/LengthSubHeader.js

```diff
@@ -0,0 +1,14 @@
+import RadioButtonsGroup from './RadioButtonsGroup';
+
+function LengthSubHeader() {
+    return(
+        <div>
+            <div>
+            <h3>Length</h3>
+            <RadioButtonsGroup />
```

Comment:
I think this component has been over refactored. If we have the `LengthSubHeader` describing the length of work experience from radios, we should just put all the radios here, no?

Reply:
yea I originally just had the radio btns, but will we never use all 4 together again?

Reply:
Hmmmm I guess we won't, but I forsee we will use a combination number of radios further in the create process: 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/37882255/121310523-4dd0cf80-c8b8-11eb-8d52-704f10b9dcd5.png">